### PR TITLE
Fixes #528 Solved the issue of misplaced Learn More Buttons in App section

### PR DESCRIPTION
**Pull request by sandeepyadav097**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

Made changes to introduce the Learn More Buttons below each application for small screens.
Edited-
image

Actual-
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -326,13 +337,16 @@ <h2>Open Event Organizer App</h2>
 					- Attendee check in through QR code scan<br>
 					- Basic event analytics
                 </p>
+                
+                <div class="row" style="margin-top:5px">
+                     <div class="col-sm-12"><a class="btn" href="https://github.com/fossasia/open-event-orga-app">Learn More</a></div>
+                </div>
+
```
Comment (NaviRocker) [inline index.html]: Remove Unwanted Lines

**Comment by vaibhavsingh97** [comment]
Can you please share deployment link

**Comment by sandeepyadav097** [comment]
@vaibhavsingh97  http://sandeepyadav.tech/gci17.fossasia.org/ Here it is

File: index.html (hunk 1 above)
Comment (sandeepyadav097) [reply index.html]: What about rest?

Comment (NaviRocker) [reply index.html]: There are many unwanted lines. Remove all

Comment (sandeepyadav097) [reply index.html]: Removed

File: index.html [hunk 2]
```diff
@@ -305,18 +305,24 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
...
-        <div class="row">
+        <div class="row text-center">
             <div class="col-sm-4">
                 <h2 class="number">Phimp.me</h2>
                 <p>
                     Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because Phimp.me is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
-                </p>
-            </div>
+                </p>                
+                <div class="row" style="margin-top:75px">
```
Comment (vaibhavsingh97) [inline index.html]: Please find another alternative to `margin-top`. you can define size of container and you can position button relative to container

File: index.html [hunk 3]
```diff
@@ -305,18 +305,24 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
...
+                    <div class="col-sm-12"><a class="btn" href="https://phimp.me">Learn More</a></div>
+                </div>                
+            </div>            
             <div class="col-sm-4">
                 <h2 class="number">PSLab Android</h2>
                 <p>
                     PSLab is a tiny pocket science lab that provides an array of equipment for doing science and engineering experiments. It can function like an oscilloscope, waveform generator, frequency counter, programmable voltage and current source and also as a data logger.
-                </p>
+                </p>                
+                <div class="row" style="margin-top:95px">
```
Comment (vaibhavsingh97) [inline index.html]: same

File: index.html [hunk 4]
```diff
@@ -326,13 +332,12 @@ <h2>Open Event Organizer App</h2>
 					- Attendee check in through QR code scan<br>
 					- Basic event analytics
                 </p>
+                <div class="row" style="margin-top:5px">
```
Comment (vaibhavsingh97) [inline index.html]: same

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Buttons are not aligned in desktop mode. PLease fix it

**Comment by sandeepyadav097** [comment]
@vaibhavsingh97  updated

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM

File: index.html [hunk 5]
```diff
@@ -304,37 +304,37 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
...
+            <div class="col-sm-4" style="height:350px">
                 <h2>Open Event Organizer App</h2>
                 <p>Open Event Organizer App provides event management features like:<br>
 					- Event manager log in and sign up<br>
 					- Manual attendee check in<br>
 					- Attendee check in through QR code scan<br>
 					- Basic event analytics
-                </p>
-                <div class="row" style="margin-top:5px">
+                </p>
```
Comment (anshumanv) [inline index.html]: Remove extra spacing.

Comment (sandeepyadav097) [reply index.html]: They were like that. I didn't change them

Comment (anshumanv) [reply index.html]: Ah, my bad, looks cool. thanks

**Review by anshumanv — APPROVED** [review]

**Review by NaviRocker — APPROVED** [review]
LGTM

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**